scripts/check-format.ts

```diff
@@ -48,4 +49,7 @@ async function checkFormat() {
   process.exit(exitCode);
 }
 
-checkFormat();
+checkFormat().catch((e) => {
+  console.error(e);
+  process.exit(1);
```

Comment:
Question: Is the only difference between this and just calling `checkFormat()` that this won't throw and unhandled promise rejection? Or are there other reasons for not just letting the error bubble up

Reply:
Functionally I'm not sure what the difference is - I think it will exit normally with code 1 instead of an "unhandled" error. Practically the difference is that if I don't put a catch on an async function call, the linter will yell at me.